db/migrate/20241211092117_create_beneficiary_addresses.rb

```diff
@@ -0,0 +1,20 @@
+class CreateBeneficiaryAddresses < ActiveRecord::Migration[8.0]
+  def change
+    create_table :beneficiary_addresses do |t|
+      t.references :account, null: false, foreign_key: true
+      t.references :beneficiary, null: false, foreign_key: { to_table: :contacts, on_delete: :cascade }
+      t.citext :iso_region_code
```

Comment:
I think we're missing `iso_country_code` here. I know it's on the beneficiary, but it doesn't really make sense to have an address without a country. So we shoulld denormalize the iso_country_code here too

Reply:
I thought about that's why I moved it to `beneficiary` because if we think about it again `address` here might not be a good name. We should call it `subscribed_regions` or something else instead of address. This table is not represent the actual addresses of beneficiary. When we think of it as a subscribed region, then we shouldn't need country repeat in there. Because it doesn't make sense for them to pass if they know it must be always be in the same country and they don't have any options to pass different as well.